python/cudf/udf_cpp/CMakeLists.txt

```diff
@@ -26,12 +26,28 @@ rapids_find_package(
   INSTALL_EXPORT_SET udf-exports
 )
 
+# The below in inspired by legate dataframe
+execute_process(
+  COMMAND "python" -c
+          "from numba.cuda.memory_management.nrt import get_include; print(get_include())"
+  OUTPUT_VARIABLE NRT_INCLUDE_DIR
+  ERROR_VARIABLE NRT_ERROR
+  RESULT_VARIABLE NRT_RESULT
+  OUTPUT_STRIP_TRAILING_WHITESPACE
+)
+if(${NRT_RESULT})
```

Comment:
Is this supposed to be `NRT_ERROR`? What does that hold compared to `NRT_RESULT`?

Reply:
No,`RESULT_VARIABLE` contains an exit code here, so we're inside this `if` on a nonzero exit code (failure). The real "result" of the call lands in `OUTPUT_VARIABLE`.